engine/src/test/java/io/camunda/zeebe/engine/util/EngineRule.java

```diff
@@ -174,8 +174,8 @@ public EngineRule withOnSkippedCallback(final Consumer<LoggedEvent> onSkippedCal
     return this;
   }
 
-  public EngineRule withReplayMode(final ReplayMode replayMode) {
-    environmentRule.withReplayMode(replayMode);
+  public EngineRule withReplayMode(final StreamProcessorMode streamProcessorMode) {
```

Comment:
(same as above) 

Align the name of the method with the parameter name. We should change it to `streamProcessorMode()`.